pkgs/development/r-modules/default.nix

```diff
@@ -412,6 +412,7 @@ let
     odbc = [ pkgs.pkg-config ];
     openssl = [ pkgs.pkg-config ];
     pdftools = [ pkgs.pkg-config ];
+    rsvg = [ pkgs.pkg-config pkgs.librsvg.dev ];
```

Comment:
Usually `pkg-config` needs to be in nativeBuildInputs also all R packages here seems wrong....

Reply:
Ah, all right. Thanks for the pointer. Re-reading the nixpkgs manual it seems like `pkg-config` should be in nativeBuildInputs (like you said), but `librsvg.dev` should stay in buildInputs (since it will still be used at runtime).
Does that sound right?

Reply:
I'm also reading some conflicting information from #76797 and #128602

Reply:
The second pr simply seems to ignore the fact that it is incorrect.